docs/interfaces/CC++.rst

```diff
@@ -25,14 +25,18 @@ The main C/C++ API is imported from the header :code:`osqp.h` and provides the f
 
 Sublevel API
 ------------
-Sublevel C/C++ API is also imported from the header :code:`osqp.h` and provides the following functions
+Sublevel C API is also imported from the header :code:`osqp.h` and provides the following functions
 
 Warm start
 ^^^^^^^^^^
 OSQP automatically warm starts primal and dual variables from the previous QP solution. If you would like to warm start their values manually, you can use
 
 .. doxygenfunction:: osqp_warm_start
 
+.. doxygenfunction:: osqp_warm_start_x
+
+.. doxygenfunction:: osqp_warm_start_y
+
```

Comment:
I thought we decided to remove these function here: https://github.com/osqp/osqp/commit/fab90f8b138aa48c9be71c1d1ddbbe45267a1308

Reply:
You can see that these function do not appear in `osqp_api_functions.h` any more.

Reply:
Done. Not changing this in master just yet.